app/javascript/packs/billboardAfterRenderActions.js

```diff
@@ -107,7 +136,10 @@ function trackAdImpression(adBox) {
   adBox.dataset.impressionRecorded = true;
 }
 
-function trackAdClick(adBox) {
+function trackAdClick(adBox, event) {
+  if (!event.target.closest('a')) {
```

Comment:
This will adjust the behavior in general so we track link clicks only — which is new but should have been the expected behavior all along.

This is added here to ensure the "closeout" click is not tracking as a link click.